contributing/workflow/bug_triage_guidelines.rst

```diff
@@ -75,17 +75,15 @@ The following labels are currently defined in the Godot repository:
    the code base.
```

Comment:
Suggest adding something about removing the label if an open PR is associated with it, to prevent duplicates as new contributors might not be able to recognize that a PR is linked, as suggested in chat

Reply:
That is a good idea. I have updated the PR.